src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -9,7 +9,7 @@ const buildBaseUrl = origin => {
 };
 
 const embedUrl = ({ origin, assetId, vpid, isAmp = false }) => {
-  const avRoute = 'ws/av-embeds';
+  const avRoute = 'ws/av-embeds/articles';
```

Comment:
Does this change not limit the MediaPlayer container to article page types only? Well artificially limiting its use based on avRoute.

Reply:
Good point, we probably want this to be a variable or we rename the function to `embedUrlArticles` because only article requests will pass a `vpid`

Reply:
Also see https://bbcnews.slack.com/archives/CHTF3GCTH/p1567496669042000

>Hey article pod. I was wondering what you thought about making the Simorgh MediaPlayer component less article and Optimo data specific so that it can be used in other page types. The other option is to rename the MediaPlayer component to MediaPlayerBlock and keep it's main responsibility to render media players of all types from blocks data.
For some more context, this is currently all I need to do to get the audio player working for live radio pages https://github.com/bbc/simorgh/pull/3514/files you can see that I'm not using the Simorgh MediaPlayer and am instead using the canonical and amp media player components from Psammead because constructing the src prop is really straightforward compared to how it is in the Simorgh MediaPlayer component https://github.com/bbc/simorgh/blob/latest/src/app/containers/MediaPlayer/index.jsx also the article page media player has specific metadata concerns.

Reply:
The current AV embed endpoint only supports Optimo assets, and as @jroebu14 pointed out the MediaPlayer container only handles Optimo blocks, this just makes that more restriction more explicit.

Reply:
@hindsc52 @jamesdonoh @dr3 what are you thoughts on moving forward? 

I'm thinking we merge as is and then as @jroebu14 has suggested we "rename the MediaPlayer component to MediaPlayerBlock and keep it's main responsibility to render media players of all types from blocks data"